gateway/src/main/java/io/zeebe/gateway/impl/configuration/AIMDCfg.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.gateway.impl.configuration;
+
+import io.zeebe.util.DurationUtil;
+import io.zeebe.util.Environment;
+import java.time.Duration;
+
+public class AIMDCfg {
```

Comment:
Can we configure the limit via environment variables?

Reply:
I can add them if it is required.